generators/new-wp-plugin/index.js

```diff
@@ -92,11 +91,33 @@ module.exports = generators.Base.extend({
       regex: 'lean-p',
       replacement: this.name
     }, settings) );
+    console.log( chalk.green('Completed!\n') );
   },
 
   install: function() {
     process.chdir( this.PLUGIN_FOLDER );
 
-    this.spawnCommandSync('composer', ['update']);
+    if ( fs.existsSync('./vendor') ) {
+      console.log( chalk.red('Composer has alredy been installed\n') );
+    } else {
+      this.spawnCommandSync('composer', ['install']);
+      this.spawnCommandSync('composer', ['update']);
+    }
+  },
+
+  end: function(){
```

Comment:
Why is this doing? If we're just creating a plugin then the .github folder should stay in the plugin route. The .github folder for the project can be created by the new-wp-project generator. Same for .travis.yml (which is already done by new-wp-project).